orttraining/orttraining/core/graph/pipeline_transformer.cc

```diff
@@ -253,12 +469,18 @@ Status TransformGraphForPipeline(
   std::string& forward_waited_output_name,
   std::string& forward_recorded_output_name,
   std::string& backward_waited_output_name,
-  std::string& backward_recorded_output_name) {
+  std::string& backward_recorded_output_name,
+  std::string& forward_waited_event_after_recv_name,
+  std::string& forward_recorded_event_before_send_name,
+  std::string& backward_waited_event_after_recv_name,
+  std::string& backward_recorded_event_before_send_name) {
   // insert WaitEvent and RecordEvent to the partition
   Node* send_fw{nullptr};
   Node* send_bw{nullptr};
   Node* recv_fw{nullptr};
   Node* recv_bw{nullptr};
```

Comment:
Nit: if we decided forward/backward has a larger scope than operator, you mind updating these naming too? It'll be nice to have unified convention across a function.

Reply:
Updated.